__tmp_kernel_none_f0_c0_r0_g0.cu

```diff
@@ -11368,11 +11369,13 @@ __global__ void __cluster_dims__(2, 1, 1) nvfuser_none_f0_c0_r0_g0(Tensor<__half
         }
       }
     }
+    return;
```

Comment:
Note: This `return` is super important, without this, our perf will be 4% of cuBLAS.